### DISCUSSION TITLE
React example?

### DISCUSSION THREAD

**Discussion by imnasnainaec** [digitalfortress-tech/typeahead-standalone#67] [Q&A] [answered]
Are there any available examples of typeahead-standalone used in a React function(al) component?

**Comment by niketpathak** [comment, marked as answer]
Hello @imnasnainaec ,

Here's a simple example of using the library with React

https://jsfiddle.net/niketpathak/34c0nvar/

The library being framework agnostic, can be used with any framework as desired.

**Reply by imnasnainaec** [reply]
Thanks, @niketpathak! The use of
```
React.useEffect(() => {
    window.typeahead({
[...]
```
is not clear in [the docs](https://typeahead.digitalfortress.tech/), so it might be helpful to add (or link to) the jsfiddle example in the docs.

**Comment by imnasnainaec** [comment]
@niketpathak I've modified the above solution to not require `typeahead-standalone` to be added/referenced outside the component where it's being used: https://codesandbox.io/s/typeahead-standalone-react-typescript-v3x4kh

**Reply by niketpathak** [reply]
That's incredible! Thanks for posting that here. Would be useful for others too.
